libraries/AP_HAL_ChibiOS/hwdef/revo-mini-bdshot/hwdef.dat

```diff
@@ -0,0 +1,14 @@
+# Bi-directional dshot version of omnibusf4pro
```

Comment:
You probably want to adjust this comment

Reply:
OK thanks.